libdebug/debugger/internal_debugger.py

```diff
@@ -163,10 +170,11 @@ def __init__(self: InternalDebugger) -> None:
         self.pprint_syscalls = False
         self.pipe_manager = None
         self.process_id = 0
-        self.threads = list()
+        self.threads = []
         self.instanced = False
         self._is_running = False
         self.resume_context = ResumeContext()
+        self._arch = map_arch(libcontext.platform)
```

Comment:
if we set the arch here with `map_arch` manually, do we really need the setter of the public property `arch`? Can we simply delete the property setter?

Reply:
Moreover, do we really need libcontext.platform?

Reply:
`platform` is the host's arch, `debugger.arch` is the debuggee's arch. These two values aren't always equal (platform=amd64, arch=i386). Additionally, `platform` is invariable, whereas the user might want to switch `arch` in some cases.